package.json

```diff
@@ -41,11 +41,13 @@
   },
   "dependencies": {
     "@patternfly/react-console": "1.x",
+    "@patternfly/react-core": "2.3.x",
```

Comment:
This might slightly complicate web-ui build which uses
```
"@patternfly/react-core": "^1.43.5"
```

Do we need 2.x now?
What about declaring that as `pperDependency`?

Reply:
not sure, but we'll certainly need it once we start using new table and inline editing

ok, I will declare it as peerDependency for now

Reply:
declared as a peerDependency: `"@patternfly/react-core": "1.43.x"`

but it will come to us again in the future